config/locales/user_mailer/en.yml

```diff
@@ -29,13 +29,11 @@ en:
         want to delete your account?
       subject: Delete your account
     account_reset_request:
-      cancel: If you don’t want to delete your account, <u>%{cancel_account_reset}</u>.
-      header: Wait 24 hours to delete your account
+      cancel: Don't want to delete your account? Sign in to your login.gov <br> account to cancel.
```

Comment:
Is the `<br>` needed?

Also we often use the `APP_NAME` constant and interpolate it via `%{app_name}` instead of hardcoding it like this

Reply:
@zachmargolis I actually used the break in this case because the automatic hyphenation was producing a kind of ugly result in the rendered email. It was producing:

> Don't want to delete your account? Sign in to your login.gov ac-
> count to cancel.

I'm certainly open to other options of overcoming that or just leaving it be. That was just the easiest thing I knew of.

Reply:
I'm always suspicious of breaks like that because on different screen sizes, font sizes, etc etc things change anyways, but that seems like a good enough reason

Reply:
Agreed! I believe in this case the email content body is a fixed width so it shouldn't run into issues with content layout changing.